release-notes/opendistro-for-elasticsearch-release-notes-1.10.1.md

```diff
@@ -368,9 +368,9 @@ Open Distro for Elasticsearch 1.10.1 includes the following features, enhancemen
 
 ### SQL
 
-* update user documentation for testing odbc driver connection on windows([#722](https://github.com/opendistro-for-elasticsearch/sql/pull/722))
-* Added workaround for identifiers with special characters in troubleshooting page([#718](https://github.com/opendistro-for-elasticsearch/sql/pull/718))
-* Update release notes for OD 1.10 release([#699](https://github.com/opendistro-for-elasticsearch/sql/pull/699))
+* update user documentation for testing odbc driver connection on windows ([#722](https://github.com/opendistro-for-elasticsearch/sql/pull/722))
```

Comment:
Do we need to capitalize the first letter of all sentences or take them as submitted?

Reply:
I just take them as submitted, it is the same procedure as before, no problems

Reply:
Please fix the grammar. "Update", "ODBC", and "Windows" should be capitalized.